main.py

```diff
@@ -6,9 +6,26 @@
 import factory
 import loader
 from shape import Rectangle, Circle, Triangle, Parallelogram, Shape
+from classes import ManageTimestamps
+from PIL import Image
+import glob
+import time
+
+print('1')
 
 os.environ["SDL_VIDEO_CENTERED"] = "1"
 
+print('2')
+
+picked_IAPS = []
```

Comment:
This never gets populated, so when you check if something exists within it further down, it's always checking against an empty list.

Reply:
Yeah, I changed my mind on using it after a while. I'll remove